### PR TITLE
Release 1.25.0

### PR DESCRIPTION
**Pull request by ywkim312**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.25.0] - 2024-02-28
+
```
Comment (ylyangtw) [inline CHANGELOG.md:9]: pyincore release PR says 2/22. Maybe should be 22?

Comment (ywkim312) [reply CHANGELOG.md:9]: oh is it? Let me ask in slack

Comment (ywkim312) [reply CHANGELOG.md:9]: Changed to today

**Review by ylyangtw — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +28,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Fixed
 - Fix Semantic search endpoint return format [#267](https://github.com/IN-CORE/incore-services/issues/267)
+- Fix broken Semantics Type embedded in data viewer [#273](https://github.com/IN-CORE/incore-services/issues/273)
```
Comment (navarroc) [inline CHANGELOG.md:31]: This is in the wrong release.

Comment (ywkim312) [reply CHANGELOG.md]: Corrected

**Review by navarroc — APPROVED** [review]

**Pull request merged**